packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.Props.ts

```diff
@@ -8,7 +8,7 @@ export interface ISearchBox {
   focus(): void;
 }
 
-export interface ISearchBoxProps extends React.Props<SearchBox> {
+export interface ISearchBoxProps extends React.HTMLAttributes<SearchBox> {
```

Comment:
hmmm shouldn't this be `HTMLAttributes<HTMLInputElement>`...

and later, mix the extra native props into the root